homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -0,0 +1,193 @@
+"""Support for Xiaomi Mi Air Quality Monitor (PM2.5)."""
+from miio import AirQualityMonitor, DeviceException
+import voluptuous as vol
+
+from homeassistant.components.air_quality import (
+    AirQualityEntity,
+    PLATFORM_SCHEMA,
+    _LOGGER,
+    ATTR_PM_2_5,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TOKEN, ATTR_TEMPERATURE
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+
+DEFAULT_NAME = "Xiaomi Miio Air Quality Monitor"
+DATA_KEY = "air_quality.xiaomi_miio"
+
+ATTR_CO2E = "carbon_dioxide_equivalent"
+ATTR_HUMIDITY = "relative_humidity"
+ATTR_TVOC = "total_volatile_organic_compounds"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MODEL = "model"
+ATTR_SW_VERSION = "sw_version"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_TOKEN): vol.All(cv.string, vol.Length(min=32, max=32)),
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    }
+)
+
+PROP_TO_ATTR = {
+    "carbon_dioxide_equivalent": ATTR_CO2E,
+    "relative_humidity": ATTR_HUMIDITY,
+    "particulate_matter_2_5": ATTR_PM_2_5,
+    "temperature": ATTR_TEMPERATURE,
+    "total_volatile_organic_compounds": ATTR_TVOC,
+    "manufacturer": ATTR_MANUFACTURER,
+    "model": ATTR_MODEL,
+    "sw_version": ATTR_SW_VERSION,
```

Comment:
We don't allow static info in state attributes. Please remove manufacturer, model and sw_version.

State attributes are for dynamic info related to the state of the entity or entity info that is essential for automations.